scripts/barclamp_install.rb

```diff
@@ -58,19 +55,41 @@ def usage()
 FileUtils.mkdir log_path unless File.directory? log_path
 log = File.join log_path, "component_install.log"
 
+install_lib_path = Pathname.new("/var/lib/crowbar/install")
+installation_steps_path = install_lib_path.join("installation_steps")
+if installation_steps_path.exist?
+  steps = installation_steps_path.exist?.map do |step|
```

Comment:
this will not work... you are mapping on a boolean here. :) probably you wanted
`installation_steps_path.readlines.map`